decidim-core/app/mailers/decidim/application_mailer.rb

```diff
@@ -17,6 +17,7 @@ def set_smtp
       return if @organization.nil? || @organization.smtp_settings.blank?
 
       mail.from = @organization.smtp_settings["from"].presence || mail.from
```

Comment:
There's a migration changing `smtp_settings["from"]` to `smtp_settings[:from_email]` in this PR, and also it is the name setted by the Form object. Shouldn't we rename it here also?
```suggestion
      mail.from = @organization.smtp_settings["from_email"].presence || mail.from
```

Reply:
resolved here 854a0c3d82a98cd2ca2305caf40f3c158ad4687a

Reply:
I rewrote the PR description, as they were some details I hadn't understood first : `from` designate what will appear in the mail sent ie. "your organization <mail@example.org>", but `from_email` designate the mail itself (in the example mail@example.org).
The migration is there "only" to be sure that the `from` won't be the default one if there is no `from_label` defined